### PR TITLE
Feature responsive design

### PR DESCRIPTION
**Pull request by waiyankyaw961999**
## Add responsive design for all devices

1. Break points are 1090px , 768px, 430px
2. Add extra backgrounds in the behind the context for medium devices and large devices

### CODE REVIEW

File: styles/work-section.css [hunk 1]
```diff
@@ -66,3 +67,19 @@
...
+    row-gap: 32px;
+  }
+}
+
+@media only screen and (max-width: 1090px) and (min-width: 768px) {
+  .work-lists {
+    grid-template-columns: repeat(2, 1fr);
+    margin: 0 auto;
+    row-gap: 32px;
+  }
```
Comment (tugsem) [inline styles/work-section.css]: - [x] In the figma design, there are six work cards but you have seven. Kindly remove one and also, the horizontal gaps between them should be smaller than you used. I am attaching the images below:
yours: 
image
figma: 
image

File: styles/info.css [hunk 2]
```diff
@@ -127,3 +127,21 @@
...
+    justify-content: center;
+  }
+}
+
+@media only screen and (max-width: 1090px) and (min-width: 768px) {
+  .info-section {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    margin: 0 auto;
+    row-gap: 32px;
```
Comment (tugsem) [inline styles/info.css:154]: - [x] Please use the same proportioning for this section of cards. Each card should have the same layout for the title and should have more padding on the list underneath:
yours:
image
figma: 
image

File: styles/contact.css [hunk 3]
```diff
@@ -96,18 +103,65 @@ textarea[type=text-area]:focus {
...
   }
 
   .contact-section {
     display: grid;
-    margin-top: 3.375rem;
-    grid-template-columns: repeat(2, 1fr);
+    height: 30.625rem;
+    margin-top: 1.09375rem;
+    grid-template-columns: repeat(6, 1fr);
+    justify-items: center;
```
Comment (tugsem) [inline styles/contact.css]: - [ ] Please use Flexbox to align items in this section. That would be more accurate. Also, please add the background image to match the figma design: 
yours: 
image
figma: 
image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @waiyankyaw961999 👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️linters are passing
✔️correct use of github flow
✔️professional documentation
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @waiyankyaw961999, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Good UI design :white_check_mark: 
- [x] GitHub flow is used correctly :white_check_mark:  
- [x] Application is deployed using GitHub pages :white_check_mark: :clap: :clap:  
- [x] Pull request has a descriptive title and summary :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**